catalog/spatial/registry/registry-schema-bindings/src/main/java/org/codice/ddf/registry/schemabindings/RegistryPackageUtils.java

```diff
@@ -0,0 +1,279 @@
+                return slot;
+            }
+        }
+
+        return null;
+    }
+
+    public static Map<String, SlotType1> getNameSlotMap(List<SlotType1> slots) {
+        Map<String, SlotType1> slotMap = new HashMap<>();
+
+        for (SlotType1 slot : slots) {
+            slotMap.put(slot.getName(), slot);
+        }
+        return slotMap;
+    }
+
+    public static Map<String, List<SlotType1>> getNameSlotMapDuplicateSlotNamesAllowed(
+            List<SlotType1> slots) {
+        Map<String, List<SlotType1>> slotMap = new HashMap<>();
+
+        for (SlotType1 slot : slots) {
+            slotMap.putIfAbsent(slot.getName(), new ArrayList<>());
+            slotMap.get(slot.getName())
+                    .add(slot);
+        }
+
+        return slotMap;
+    }
+
+    public static List<String> getSlotStringValues(SlotType1 slot) {
+        List<String> slotAttributes = new ArrayList<>();
+
+        if (slot.isSetValueList()) {
+            ValueListType valueList = slot.getValueList()
+                    .getValue();
+            if (valueList.isSetValue()) {
+                slotAttributes = valueList.getValue();
+            }
+        }
+
+        return slotAttributes;
+    }
+
+    public static List<Date> getSlotDateValues(SlotType1 slot) {
+        List<Date> dates = new ArrayList<>();
+
+        if (slot.isSetValueList()) {
+            ValueListType valueList = slot.getValueList()
+                    .getValue();
+
+            if (valueList.isSetValue()) {
+                List<String> values = valueList.getValue();
+
+                for (String dateString : values) {
+                    Date date = Date.from(ZonedDateTime.parse(dateString)
+                            .toInstant());
+                    if (date != null) {
+                        dates.add(date);
+                    }
+                }
+            }
+        }
+
+        return dates;
+    }
+
+    public static String getStringFromIST(InternationalStringType internationalString) {
+        String stringValue = "";
+        List<LocalizedStringType> localizedStrings = internationalString.getLocalizedString();
+        if (CollectionUtils.isNotEmpty(localizedStrings)) {
+            LocalizedStringType localizedString = localizedStrings.get(0);
+            if (localizedString != null) {
+                stringValue = localizedString.getValue();
+            }
+        }
+
+        return stringValue;
+    }
+
+    public static InternationalStringType getInternationalStringTypeFromString(
+            String internationalizeThis) {
+        InternationalStringType ist = RIM_FACTORY.createInternationalStringType();
+        LocalizedStringType lst = RIM_FACTORY.createLocalizedStringType();
+        lst.setValue(internationalizeThis);
+        ist.setLocalizedString(Collections.singletonList(lst));
+
+        return ist;
+    }
+
+    public static SlotType1 getSlotFromString(String slotName, String slotValue, String slotType) {
```

Comment:
Should we name those "factory" methods something like `create...()` instead? So here we would have two overloaded methods called `createSlot()` instead of two methods with different names.